src/navigation/NavItem.jsx

```diff
@@ -0,0 +1,130 @@
+import PropTypes from 'prop-types';
```

Comment:
Shouldn't this component have a story too??

Reply:
No, according to how they want it built, NavItem is not going to be something that is imported into the project, strictly NavBar.